docs/docs/how-to/performance/improving-site-performance.md

```diff
@@ -75,11 +75,11 @@ For slightly more effort, you can get additional performance gains; rather than
 
 There are a number of places to put an inlined script, depending whether you need it to execute immediately upon loading or can defer execution.
 
-- _No deferring_: This is a good default. Put the script in [onPreRenderHTML](https://www.gatsbyjs.org/docs/ssr-apis/#onPreRenderHTML) to have it added to your document tree. You can place it lower in your DOM to have parsed and evaluated later.
-- _Some deferring_: You can place the script in [onClientEntry](https://www.gatsbyjs.org/docs/browser-apis/#onClientEntry) to have it execute after page load, but before the browser renders the page.
-- _More deferring_: You can place the script in [onInitialClientRender](https://www.gatsbyjs.org/docs/browser-apis/#onInitialClientRender) to have it execute after the browser renders the page.
+- _No deferring_: This is a good default. Put the script in [onPreRenderHTML](/docs/ssr-apis/#onPreRenderHTML) to have it added to your document tree. You can place it lower in your DOM to have parsed and evaluated later.
```

Comment:
Those links are still relying on redirects, the correct one here would be: https://www.gatsbyjs.com/docs/reference/config-files/gatsby-ssr/#onPreRenderHTML

Or in the MD file then: `/docs/reference/config-files/gatsby-ssr/#onPreRenderHTML`